pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -239,6 +231,24 @@ stdenv.mkDerivation {
       fi
     ''
 
+    + optionalString (cc ? gcc && isClang) ''
+      ##
+      ## General Clang support
+      ##
+
+      echo "-target ${targetPlatform.config}" >> $out/nix-support/cc-cflags
+    ''
+
+    + optionalString (cc ? gcc && isClang && libcxx == null) ''
+      ##
+      ## GCC libs for non-GCC support
+      ##
+
+      echo "-B${cc.gcc.cc}/lib/gcc/${targetPlatform.config}/${cc.gcc.version}" >> $out/nix-support/cc-cflags
```

Comment:
does `/lib/gcc/${targetPlatform.config}` exists on native builds?

Reply:
I think it does as of recently.